uber/custom_tags.py

```diff
@@ -445,12 +445,11 @@ def render(self, context):
 class single_day_prices(template.Node):
     def render(self, context):
         prices = ''
-        skiprest = ''
         for day, price in BADGE_PRICES['single_day'].items():
             if day == datetime.strftime(ESCHATON, "%A"):
                 prices += 'and ${} for {}'.format(price, day)
-                skiprest = 'YES'
-            elif skiprest == '':
+                break
+            else
```

Comment:
This totally does get used, specifically in the prereg_closed page. @earl7399 just based this off the wrong branch.